en/user-guide/layerexplanations/layerexplanation.rst

```diff
@@ -27,33 +32,33 @@ Main layer for line-wastewater structures (channels). Creating a new record in t
 Wastewater Structures
 ---------------------
 
-od_wastewater_structure
-^^^^^^^^^^^^^^^^^^^^^^^
+Wastewater Structures (od_wastewater_structure)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 This layer shows the detailed geometries of wastewater structures. You digitize a new detailed geometry by the records of the layer vw_qgep_wastewater_structure. If you want to change an existing geometry, then you have to edit in the layer od_wastewater_structure. In database-schema qgep_od.
 
-Structure Parts
-^^^^^^^^^^^^^^^
+Structure Parts (od_structure_part)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Cover is the only structure part with a point-geometry himself. All other structure parts are just linked to there wastewater structures and should by only edited by the main layers (vw_qgep_wastewater_structure and vw_qgep_reach).
 
-vw_cover
-^^^^^^^^
+Covers (vw_cover)
+^^^^^^^^^^^^^^^^^
 
 You use this layer, if you want to change the situation of just one selected cover (and not the hole wastewater structure) or if you want to add a new cover to an existing wastewater structure. (you can add an additional cover also in the covers-tab of the vw_qgep_wastewater_structure, but this cover has no situation...). And you use this layer if you want to show the detail position of the covers (e.g. in network_plan or pipeline_registry) or to export the cover positions. In database-schema qgep_od.
 
-vw_channel
-^^^^^^^^^^
+Channels (vw_channel)
+^^^^^^^^^^^^^^^^^^^^^
 
 The channel-class has no geometry and is therefore mostly changed in the vw_qgep_reach layer. In database-schema qgep_od.
 
-od_organisation
+Organisations (od_organisation)
 ^^^^^^^^^^^^^^^
 
 todo: This table is today at a strange place (it is not only related to wastewater structures) and has to be revised. In database-schema qgep_od.
 
-Inspection
-----------
+Inspections
+-----------
 
 These tables are used in the maintenance tabs in the main tables. Do not edit directly. In database-schema qgep_od.
```

Comment:
Do not edit directly is true - but maybe we should give a reason for this.

Reply:
I think the inspection / maintenance should be documented in the maintenance section. A short sentence could refer to it.

I can either let a todo section or add something to explain why we should not edit it directly.

Reply:
Where is this inspection /maintenance section?

Reply:
https://qgep.github.io/docs/user-guide/editing/maintenance_events.html

Actually we start from reaches and add a maintenance event, but the workflow could be reverted too. 

Ex :
1. A maintenance event is planned and entered into the system.
2. Affected objects are selected from the maintenance event as they are impacted by it.

Same scenario is valid for an inspection too.

But here we should refer to the documentation about maintenance events when saying it can or iz cannot be used directly.

Reply:
Then just link to that documentation.

Reply:
Link created, I'll open a new PR to fix some incoherences around the maintenance section.